examples/with-google-tag-manager/lib/gtm.js

```diff
@@ -0,0 +1,8 @@
+export const GTM_ID = process.env.NEXT_PUBLIC_GOOGLE_TAG_MANAGER_ID
+
+export const pageview = (url) => {
+  window.dataLayer({
```

Comment:
`window.dataLayer` is not a method. Did you actually mean to use `window.dataLayer.push` instead?